packages/wrangler/src/pages/publish.tsx

```diff
@@ -459,3 +437,31 @@ export const Handler = async ({
 	);
 	await metrics.sendMetricsEvent("create pages deployment");
 };
+
+async function promptUseNewOrExistingProject(): Promise<"new" | "existing"> {
+	return select(
+		"No project selected. Would you like to create one or use an existing project?",
+		[
+			{
+				label: "Create a new project",
+				value: "new",
+			},
+			{
+				label: "Use an existing project",
+				value: "existing",
+			},
+		],
+		0
+	) as Promise<"new" | "existing">;
```

Comment:
Instead of having to cast, we might consider updating the type of `select(...)` to infer the string union `'new' | 'existing'`

<img width="493" alt="image" src="https://user-images.githubusercontent.com/767070/209192055-bc257997-ee05-46d8-9825-ec3d842ea079.png">